src/test/java/com/ounitech/wemove/repositories/EntryRepositoryTest.java

```diff
@@ -6,13 +6,15 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
 import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.TestPropertySource;
 
 import java.util.List;
 import java.util.function.Function;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @DataJpaTest
+@TestPropertySource(locations = "classpath:application-test.properties")
```

Comment:
not needed as discussed

Reply:
The @DataJpaTest annotation sets up a default embedded database, so there's no need to add a test environment to the application. My question is: is it better to override the default setup by adding a test environment, or we can rely on the @DataJpaTest configuration and remove the application-test.properties file?

Reply:
you can remove application-test.properties. @DataJpaTest defaults suits our needs for now